boards.txt

```diff
@@ -15986,6 +15986,256 @@ heltec_wireless_stick_lite.menu.EraseFlash.none.upload.erase_cmd=
+heltec_wireless_tracker.build.board=heltec_wireless_tracker
+
+heltec_wireless_tracker.build.usb_mode=1
+heltec_wireless_tracker.build.cdc_on_boot=0
+heltec_wireless_tracker.build.msc_on_boot=0
+heltec_wireless_tracker.build.dfu_on_boot=0
+heltec_wireless_tracker.build.f_cpu=240000000L
+heltec_wireless_tracker.build.flash_size=4MB
+heltec_wireless_tracker.build.flash_freq=80m
+heltec_wireless_tracker.build.flash_mode=dio
+heltec_wireless_tracker.build.boot=qio
+heltec_wireless_tracker.build.boot_freq=80m
+heltec_wireless_tracker.build.partitions=default
+heltec_wireless_tracker.build.loop_core=
+heltec_wireless_tracker.build.event_core=
+heltec_wireless_tracker.build.psram_type=qspi
+heltec_wireless_tracker.build.memory_type={build.boot}_{build.psram_type}
+
+heltec_wireless_tracker.menu.PSRAM.disabled=Disabled
+heltec_wireless_tracker.menu.PSRAM.disabled.build.psram_val=
+heltec_wireless_tracker.menu.PSRAM.disabled.build.psram_type=qspi
+heltec_wireless_tracker.menu.PSRAM.enabled=QSPI PSRAM
+heltec_wireless_tracker.menu.PSRAM.enabled.build.psram_val=-DBOARD_HAS_PSRAM
+heltec_wireless_tracker.menu.PSRAM.enabled.build.psram_type=qspi
+heltec_wireless_tracker.menu.PSRAM.opi=OPI PSRAM
+heltec_wireless_tracker.menu.PSRAM.opi.build.psram_val=-DBOARD_HAS_PSRAM
+heltec_wireless_tracker.menu.PSRAM.opi.build.psram_type=opi
+
+heltec_wireless_tracker.menu.FlashMode.qio=QIO 80MHz
+heltec_wireless_tracker.menu.FlashMode.qio.build.flash_mode=dio
+heltec_wireless_tracker.menu.FlashMode.qio.build.boot=qio
+heltec_wireless_tracker.menu.FlashMode.qio.build.boot_freq=80m
+heltec_wireless_tracker.menu.FlashMode.qio.build.flash_freq=80m
+heltec_wireless_tracker.menu.FlashMode.qio120=QIO 120MHz
+heltec_wireless_tracker.menu.FlashMode.qio120.build.flash_mode=dio
+heltec_wireless_tracker.menu.FlashMode.qio120.build.boot=qio
+heltec_wireless_tracker.menu.FlashMode.qio120.build.boot_freq=120m
+heltec_wireless_tracker.menu.FlashMode.qio120.build.flash_freq=80m
+heltec_wireless_tracker.menu.FlashMode.dio=DIO 80MHz
+heltec_wireless_tracker.menu.FlashMode.dio.build.flash_mode=dio
+heltec_wireless_tracker.menu.FlashMode.dio.build.boot=dio
+heltec_wireless_tracker.menu.FlashMode.dio.build.boot_freq=80m
+heltec_wireless_tracker.menu.FlashMode.dio.build.flash_freq=80m
+heltec_wireless_tracker.menu.FlashMode.opi=OPI 80MHz
+heltec_wireless_tracker.menu.FlashMode.opi.build.flash_mode=dout
+heltec_wireless_tracker.menu.FlashMode.opi.build.boot=opi
+heltec_wireless_tracker.menu.FlashMode.opi.build.boot_freq=80m
+heltec_wireless_tracker.menu.FlashMode.opi.build.flash_freq=80m
+heltec_wireless_tracker.build.defines=-D{build.band} {build.psram_val} -DLoRaWAN_DEBUG_LEVEL={build.LoRaWanDebugLevel} -DACTIVE_REGION=LORAMAC_{build.band} -DLORAWAN_PREAMBLE_LENGTH={build.LORAWAN_PREAMBLE_LENGTH} -DLORAWAN_DEVEUI_AUTO={build.LORAWAN_DEVEUI_AUTO} -D{build.board}
+heltec_wireless_tracker.build.extra_libs=-lheltec_{build.cdc_on_boot}
+
+heltec_wireless_tracker.menu.EraseFlash.none=Disabled
```

Comment:
build.board.defines already exist at the end of board definition.
Erase flash menu is not complete and please add it to the end of board definition as other boards do (check next comment)
```suggestion

```